cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,148 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 6.2.0 <http://docs.cask.co/cdap/6.2.0/index.html>`__
+=============================================================
+
+Summary
+-------
+
+This release introduces a number of new features, improvements, and bug fixes to CDAP. Some of the main highlights of the release are:
+
+1. **Replication**
+    - Easily replicate data from databases through change data capture.
+
+2. **Google Cloud Dataproc Runtime Improvement**
+    - Google Cloud Dataproc runtime incorporates native Google Cloud Dataproc's API for job submission.
+
```

Comment:
3. Pipeline Studio Improvements:
Added the ability to perform bulk operations (copy, delete) in the pipeline Studio. Also added a right-click context menu for the Studio.